R/rules-replacement.R

```diff
@@ -0,0 +1,14 @@
+force_assignment_op <- function(pd) {
+  to_replace <- pd$token == "EQ_ASSIGN"
```

Comment:
I think it is safe, at least one test indicates that, not all `=` are replaced with `<-`.
```r
a <- 3
data_frame(a = 3)
```
However, I think you are right, maybe we should add more options to `get_transformers_nested()` so the user has more control over which transformer functions are included. What do you think?